src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        bulldozers.add(new Bulldozer("Mull", "Black", 200));
+        bulldozers.add(new Bulldozer("Donk", "Pink", 300));
+        bulldozers.add(new Bulldozer("Kong", "Yellow", 400));
+        return bulldozers;
```

Comment:
You can return immediately using List.of(new Bulldozer("Mull", "Black", 200), ... )

Reply:
Thank you, done